keyboards/clawsome/sedan/config.h

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 AAClawson (AlisGraveNil)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x7767
+#define PRODUCT_ID      0x8C78
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AlisGraveNil
+#define PRODUCT         The Sedan
+#define DESCRIPTION     A QMK-powered 68% custom keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B4 }
+#define MATRIX_COL_PINS { B5, B6, B7, C0, C1, C2, C3, C4, C5, C6, C7, D0, D1, D2, D3 }
```

Comment:
@AlisGraveNil Is this accurate to an elite-C? I'm debugging a fully built board that has an issue with multiple key-downs and I've never seen a pinout diagram of an elite C look like this (no F pins, C0?). 

Can open a separate issue if this is truly off, but I'm trying to figure out if I'm just going crazy or not.

Reply:
To clarify, based on The Sedan v1.2 that's available

Reply:
I have no idea where it's getting 'C0' from - that's not at all the pinout I created last year.

Do you have a Sedan on hand that's bugging out?

Reply:
I do! Happy to assist.

Reply:
email me (alex@clawboards.xyx) and I'll send you a working firmware that I've had archived, and I'll look at doing a fork for fixes in a bit